scribble-lib/scribble/xelatex-render.rkt

```diff
@@ -0,0 +1,17 @@
+#lang scheme/base
```

Comment:
Same here, can you try re-using code from `pdf-render.rkt`? There's like a 6-character diff between the `*-render.rkt` files.

(This is just my opinion, trying to give constructive suggestions until someone with authority to say "yes/no to merge" reviews.)